test/e2e/all_in_one_test.go

```diff
@@ -34,7 +34,7 @@ type AllInOneTestSuite struct {
 }
 
 func(suite *AllInOneTestSuite) SetupSuite() {
-	t := suite.T()
+	t = suite.T()
```

Comment:
Are these still required, as it is related to the suite setup before any test is run - and therefore the `SetupTest` would be called before that.

Reply:
@objectiser Yes, that is required there.  SetupSuite is only called once per suite, and is actually called before SetupTest is ever called.  (SetupTest is called once per test case)

Reply:
Yes understand - what I meant is, will `t` be used between the SetupSuite being called and the first SetupTest being called - if not, then the instantiation in SetupSuite is redundant.

Reply:
t is used in setupSuite() for the call to prepare() (which creates the namespaces) and for both of the require calls.

Reply:
Ok thanks for the clarification.